heatmapchart/src/utils/get-formatted-axis-label.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { formatWithTimeZone } from '@perses-dev/components';
+
+// https://echarts.apache.org/en/option.html#xAxis.axisLabel.formatter
+export function getFormattedStatusHistoryAxisLabel(rangeMs: number, timezone: string) {
```

Comment:
;)

```suggestion
export function getFormattedHeatmapAxisLabel(rangeMs: number, timezone: string) {
```

Btw couldn't we have a shared package in this repo for such utilities to avoid copy-pasting?

Reply:
We could probably, but it's way easier and faster to copy.